README.md

```diff
@@ -34,12 +34,18 @@ class App extends React.Component {
         this.setState({ visible: false });
     }
 
+    onKeyboardKeyPress = (event) => {
+        if (event.which === 13) { // Closing on only clicking on Enter
+            this.hide();
+        }
+    }
```

Comment:
This part should be removed. ^_^

Reply:
If it is removed then any key press might be treated as event for hiding modal.
We can add another optional prop lilke `exitOnKeyPress` which will take the keyCode. We can set it default to`13`. That way if user is providing the select with spacebar button then it can be handled.

Reply:
I mean this document should be removed, not source code.

Reply:
Oh ok. got it. Removing it👍

Reply:
Removed comment.